lib/translator/translation.h

```diff
@@ -18,6 +18,7 @@
 #include "absl/types/span.h"
 
 #include "common.h"
+#include "request_sense.h"
```

Comment:
Best practice to include implementation dependent headers in .cc whenever possible